outlook/src/taskpane/components/Project/TasksSection/SelectProjectDropdown.tsx

```diff
@@ -0,0 +1,171 @@
+                }
+            });
+        }
+
+    }
+
+    private createProject = () => {
+
+        const createProjectRequest = sendHttpRequest(HttpVerb.POST,
+            api.baseURL + api.createProject, ContentType.Json,
+            this.context.getConnectionToken(), {name: this.state.query}, true);
+
+        this.setState({isLoading: true});
+
+        createProjectRequest.promise.then((response) => {
+            const parsed = JSON.parse(response);
+            const createdProject = Project.fromJson(parsed.result);
+            this.props.onProjectClick(createdProject);
+        }).catch((error) => {
+            if (error) {
+                this.setState({isLoading: false, projects: []});
+                this.context.showHttpErrorMessage(error);
+                this.setState({isLoading: false});
+            }
+        });
+    }
+
+    render() {
+
+        let searchBar = (
+            <div className="project-search-bar">
+                <TextField className="input-search" placeholder={_t('Search Projects...')}
+                           onChange={this.onQueryChanged} value={this.state.query} autoComplete="off"
+                           onFocus={(e) => e.target.select()}
+                />
+            </div>
+        );
+
+        let projects = null;
+        if (this.state.isLoading) {
+            projects = (
+                <Spinner theme={OdooTheme}
+                         size={SpinnerSize.large}
+                         className="project-result-spinner"/>
+            );
+        } else {
+            let createProjectDiv = (
+                <div className="create-project-text" onClick={() => {
+                    this.createProject()
+                }}>
+                    {_t("Create %(name)s", {name: this.state.query})}
+                </div>
+            );
+            if (this.state.projects.length > 0) {
+                if (this.state.projects.filter(project =>
+                    project.name.toUpperCase() === this.state.query.toUpperCase()).length > 0) {
+                    createProjectDiv = null;
+                }
+
+                let projectsList = this.state.projects.map(project => {
+                    return (
+                        <div key={project.id} className="project-search-result-text"
+                             onClick={() => this.props.onProjectClick(project)}>
+                            {project.name}
+                        </div>
+                    )
+                })
+                projects = (
+                    <div>
+                        {projectsList}
+                        {createProjectDiv}
+                    </div>);
+            } else {
+                if (this.state.query.length > 0) {
+                    projects = (
+                        <div>
+                            {createProjectDiv}
+                        </div>
+                    );
+                }
+            }
+        }
+
+        return (
+            <>
```

Comment:
What is this ``<>`` ?

Reply:
This is used to specify a React Fragment.